templates/lxterminal.dot

```diff
@@ -0,0 +1,20 @@
+[general]
+bgcolor=rgb({{=c.background}})
+color_preset=Custom
```

Comment:
Could this be called Srcery, or is this related to some inheritence?

Reply:
Yes, I'll change that.
But in the preferences GUI it always says "Custom" if a non-built-in theme is used.

Reply:
I see.  But just for concistany. 😉 

Need to either rerun or change output file as well.